charts/federation-v2/charts/controllermanager/templates/federationconfig.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: core.federation.k8s.io/v1alpha1
+kind: FederationConfig
+metadata:
+  name: federation-v2
+  namespace: {{ .Release.Namespace }}
+spec:
+  limited-scope: {{ .Values.global.limitedScope }}
+{{- if .Values.registryNamespace }}
+  registry-namespace: {{ .Values.registryNamespace }}
+{{- else }}
+{{- if .Values.global.limitedScope }}
+  registry-namespace: {{ .Release.Namespace }}
+{{- else }}
+  registry-namespace: kube-multicluster-public
+{{- end }}
+{{- end }}
+  controller-duration:
+{{- if .Values.clusterAvailableDelay }}
```

Comment:
Please refer to helm best practice for setting defaults:

https://github.com/helm/helm/blob/master/docs/chart_best_practices/values.md#flat-or-nested-values

Reply:
done